services/src/main/java/org/jboss/windup/web/services/rest/AnalysisContextEndpoint.java

```diff
@@ -25,6 +27,11 @@
     @Path("{id}")
     AnalysisContext get(@PathParam("id") Long id);
 
+
+    @PUT
+    @Path("storeDefaultConfigForProject/{projectId}")
+    AnalysisContext storeDefaultConfigForProject(@Valid AnalysisContext analysisContext, @PathParam("projectId") @NotNull Long projectId);
+
```

Comment:
The task was, use either default config, or, if it doesn't exist, the last run. This PR is the implementation of that. Is this change of the task?